network/address_test.go

```diff
@@ -398,7 +412,6 @@ var selectInternalTests = []selectTest{{
 		network.NewScopedAddress("2001:db8::1", network.ScopePublic),
 		network.NewScopedAddress("fc00::1", network.ScopeCloudLocal),
 		network.NewScopedAddress("8.8.8.8", network.ScopePublic),
-		network.NewScopedAddress("10.0.0.1", network.ScopeCloudLocal),
```

Comment:
Because otherwise it is selected in preference to the IPv6 one and its presence isn't relevant to that specific test.